src/visualization/streamlit-exp-v2.py

```diff
@@ -0,0 +1,108 @@
+'''
```

Comment:
Why do you have two files named `streamlit-exp-<something>`? This is your *source* code. You should note have experimental scripts here. Only put the streamlit files you plan to use for your frontend here. And give them better names. Nothing in source code should ever be an experiment!

Reply:
In the most recent commit, I updated this to reflect a better naming convention. The v1 script (which was more experimentation than v1) was relocated to the `experiments` folder, while the v2 script was updated to `streamlit-v1.py` within the `src/visualization` folder.